components/pkg-export-kubernetes/plan.sh

```diff
@@ -0,0 +1,79 @@
+pkg_name=hab-pkg-export-kubernetes
+_pkg_distname=$pkg_name
+pkg_origin=core
+pkg_version="0.1.0"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('Apache-2.0')
+pkg_build_deps=(
+  core/musl core/zlib-musl core/xz-musl core/bzip2-musl core/libarchive-musl
+  core/openssl-musl core/libsodium-musl
+  core/coreutils core/rust core/gcc
+)
+pkg_bin_dirs=(bin)
+
+bin=$_pkg_distname
+
+_common_prepare() {
```

Comment:
Frankly speaking - this is a copy pasta of the plan.sh from the new docker exporter. I took it because the kubernetes exporter is a similar kind of component to docker exporter. So that's probably why this plan is highly magical (to me at least).

I'd be grateful, if somebody had a look at the plan and tell me what stuff there is really required and what can be dropped.

Reply:
@krnowak Since I made this comment, I was poking around in a bunch of our other plans and found these same patterns. I have the same questions, but now no longer about this PR 😄 

This is fine for now. We (the Habitat team) need to take a pass through all our plans to modernize them anyway, so we can do the cleanup then.